libs/app-utils/models/src/lib/msc-harmony.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import {
+  AppFileSystemGuardOptions,
+  AppFileSystemScope,
+} from "./app-file-system"
+
+export const MSC_HARMONY_SCOPE: AppFileSystemScope = "userData" as const
+export const MSC_HARMONY_SCOPE_CATALOG_DIR = "msc-harmony" as const
+
+export const getMscHarmonyLocation = (
+  fileName?: string
+): AppFileSystemGuardOptions => {
+  return {
+    scopeRelativePath: fileName
+      ? [MSC_HARMONY_SCOPE_CATALOG_DIR, fileName]
+      : MSC_HARMONY_SCOPE_CATALOG_DIR,
+    scope: MSC_HARMONY_SCOPE,
+  }
+}
```

Comment:
`models` library is strictly for types and interfaces. Please do not export any functions from it. If the function should be available for both main and renderer processes, export it from a library with "type:utils" tag, in this case `app-utils/common`